3.0.0/README.md

```diff
@@ -0,0 +1,327 @@
+# TileJSON 3.0.0
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+**Table of contents**
+
+1. [Purpose](#1-purpose)
+1. [File format](#2-file-format)
+1. [Structure](#3-structure)
+   1. [tilejson](#31-tilejson)
+   1. [tiles](#32-tiles)
+   1. [vector_layers](#33-vector_layers)
+   1. [attribution](#34-attribution)
+   1. [bounds](#35-bounds)
+   1. [center](#36-center)
+   1. [data](#37-data)
+   1. [description](#38-description)
+   1. [fillzoom](#39-fillzoom)
+   1. [grids](#310-grids)
+   1. [legend](#311-legend)
+   1. [maxzoom](#312-maxzoom)
+   1. [minzoom](#313-minzoom)
+   1. [name](#314-name)
+   1. [scheme](#315-scheme)
+   1. [template](#316-template)
+   1. [version](#317-version)
+1. [Examples](#4-examples)
+1. [Caching](#5-caching)
+
+# 1. Purpose
+
+This specification attempts to create a standard for representing metadata about multiple types of web-based map layers, to aid clients in configuration and browsing.
+
+# 2. File Format
+
+TileJSON manifest files use the JSON format as described in [RFC 8259](https://tools.ietf.org/html/rfc8259).
+
+# 3. Structure
+
+The following describes the structure of a TileJSON object. Implementations MUST treat unknown keys as if they weren't present. However, implementations MUST expose unknown key value pairs so users can optionally handle these keys. Implementations MUST treat invalid values for keys as if they weren’t present. If the the key is optional and the value is invalid, the default value MAY be applied. If the key is required, implementations MUST treat the entire TileJSON manifest file as invalid and refuse operation.
+
+*The word "implementation" in the following sections refers to services or tools that serve, generate, or validate TileJSON objects.*
+
+## 3.1 `tilejson`
+
+REQUIRED.
+
+A semver.org style version number as a string. Describes the version of the TileJSON spec that is implemented by this JSON object.
+
+```JSON
+{
+  "tilejson": "3.0.0"
+}
+```
+
+## 3.2 `tiles`
+
+REQUIRED.
+
+An array of tile endpoints. {z}, {x} and {y}, if present, are replaced with the corresponding integers. If multiple endpoints are specified, clients may use any combination of endpoints. All endpoint urls MUST be absolute. All endpoints MUST return the same content for the same URL. The array MUST contain at least one endpoint. The tile extension is NOT limited to any particular format. Some of the more popular are: mvt, vector.pbf, png, webp, and jpg.
+
+```JSON
+{
+  "tiles": [
+    "http://localhost:8888/admin/1.0.0/world-light,broadband/{z}/{x}/{y}.mvt"
+  ]
+}
+```
+
+## 3.3 `vector_layers`
+
+REQUIRED.
```

Comment:
Elsewhere it says jpeg is a valid format, so presumably that's not the intention. Presumably it should only be required if using a vector format. But there is nowhere that explicitly says whether the contents are raster or vector (or something else?), and perhaps there should be. Otherwise the consumer has to infer that .mvt and .geojson are vector formats etc.

Reply:
I originally written it as `REQUIRED (for vector tiles)` which didn't feel like the perfect solution https://github.com/mapbox/tilejson-spec/pull/42#discussion_r186890251 & https://github.com/mapbox/tilejson-spec/pull/42/files/6b8b5ad106ad054bbec786b1a4f1420cda24fde8#r188836445

But maybe it's better than being vague? I think including a higher-level field describing vector/raster tiles will be a great improvement for v4. Going to ticket that out.